docs/releases/1.9.txt

```diff
@@ -416,6 +416,19 @@ Miscellaneous
   Therefore, the ``@skipIfCustomUser`` decorator is no longer needed to
   decorate tests in ``django.contrib.auth``.
 
+* Changes were made a few methods in generic Class Based Views to reduce
+  interdependencies between methods.
+  :meth:`SingleObjectMixin.get_context_data <django.views.generic.detail.SingleObjectMixin.get_context_data>`
+  and
+  :meth:`MultipleObjectMixin.get_context_data <django.views.generic.list.MultipleObjectMixin.get_context_data>`
+  will now get the object or object list from respective methods if they are
+  not set in ``get()`` or ``post()`` methods.
+  :meth:`SingleObjectTemplateResponseMixin.get_template_names <django.views.generic.detail.SingleObjectTemplateResponseMixin.get_template_names>`
+  and
+  :meth:`MultipleObjectTemplateResponseMixin.get_template_names <django.views.generic.list.MultipleObjectTemplateResponseMixin.get_template_names>`
+  will also no longer raise an exception if the object or object list are not
```

Comment:
Are the possible exceptions thrown all known? If so, would listing them here be useful?